control/pid_longitudinal_controller/include/pid_longitudinal_controller/pid_longitudinal_controller.hpp

```diff
@@ -94,6 +94,7 @@ class PidLongitudinalController : public trajectory_follower::LongitudinalContro
   nav_msgs::msg::Odometry m_current_kinematic_state;
   geometry_msgs::msg::AccelWithCovarianceStamped m_current_accel;
   autoware_auto_planning_msgs::msg::Trajectory m_trajectory;
+  autoware_auto_vehicle_msgs::msg::ControlModeReport m_current_control_mode;
```

Comment:
Let me think about whether `control mode report` is the best one to know the current state is manual or autonomous.

internal link
https://star4.slack.com/archives/C4P0NSMB5/p1673340967431229

Reply:
@takayuki5168 Thank you. It is addressed in https://github.com/autowarefoundation/autoware.universe/pull/2619/commits/2f0948954fffdf65174210bc93eea5244b77da68 : `ControlModeReport` -> `OperationModeState`